docs/api/jinja2.md

```diff
@@ -1,50 +1,63 @@
 # Jinja2 templates
 
-Django-Flags provides template functions that can be added to a Jinja2 environment and subsequently used in templates.
+Django-Flags provides an extension for adding the template tags to your Jinja2
+backend configuration to enable the tags to be used in Jinja2 templates.
 
 ```python
-from flags.template_functions import (
-    flag_enabled,
-    flag_disabled
-)
-from jinja2 import Environment
-
-...
-
-env = Environment(…)
-env.globals.update(
-    flag_enabled=flag_enabled,
-    flag_disabled=flag_disabled
-)
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        ...
+    },
+    {
+        'BACKEND': 'django.template.backends.jinja2.Jinja2',
+        ...
+        'OPTIONS': {
+            'extensions': [
+                ...
+                'flags.jinja2tags.flags',  # add this line to your existing settings
+                ...
+            ],
+        }
+    },
+]
 ```
 
+
 ## Checking state
 
 ### `flag_enabled`
 
-Returns `True` if a flag is enabled by for the given request, otherwise returns `False`.
+Returns `True` if a flag is enabled for the current request, otherwise returns `False`.
```

Comment:
It's entirely possible a template could be rendered outside of a context where a request is available. Instead of saying "for the current request", it might better to remove that clause entirely and add another sentence about how if a request exists in the current context it will be passed to any conditions that use it. 

We should probably do the same for the Django template documentation too.